flask/homework.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+
+#vegetables = ['beans', 'carrot', 'beetroot', 'cucumber']
```

Comment:
delete commented code or in this case, you should name variables as constants and use tuple instead of a list